src/components/FileComparison.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { FileComparisonTypeIconContainer, FileComparisonContainer } from './FileComparison.style';
+import DocIcon from './icons/DocIcon';
+import MovIcon from './icons/MovIcon';
+import Mp3Icon from './icons/Mp3Icon';
+import PngIcon from './icons/PngIcon';
+
+export default function FileComparison(): JSX.Element {
```

Comment:
This should take an input string for the text to use rather than relying on Lorem Ipsum.

Reply:
It should also take a variable for the icon type (i.e. PngIcon, AudioIcon, DocIcon assuming those are the names)

Reply:
Updated the `FileComparison` component last night to include the correct text/icons